    computed_torque_controller:
    type: panda_controllers/ComputedTorque
    arm_id: $(arg prefix)arm
    joint_names:
         - $(arg prefix)arm_joint1
         - $(arg prefix)arm_joint2
         - $(arg prefix)arm_joint3
         - $(arg prefix)arm_joint4
         - $(arg prefix)arm_joint5
         - $(arg prefix)arm_joint6
         - $(arg prefix)arm_joint7
    # --- Controller parameters --- #
    kp1: 300.0   #default: 120.0
    kp2: 150.0   #default: 70.0
    kp3: 30.0    #default: 50.0
    kv1: 30.0
    kv2: 15.0
    kv3: 3.0
    dt: 5

computed_torque_controller_mod:
    type: panda_controllers/ComputedTorqueMod
    arm_id: panda
    joint_names: *jointNames
    
    # --- Controller parameters --- #
    kp1: 300.0   #default: 120.0
    kp2: 150.0   #default: 70.0
    kp3: 30.0    #default: 50.0
    kv1: 30.0
    kv2: 15.0
    kv3: 3.0
    dt: 0.001
    gainRlinks: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01]                   # gain for R matrix
    gainRparam: [1, 0.1, 0.01]                          # gain proportional to [mass, mass*length, mass*length^2] 
    update_param: false
   

cartesian_impedance_controller_softbots:
    type: panda_controllers/CartesianImpedanceControllerSoftbots
    arm_id: $(arg prefix)arm
    joint_names:
        - $(arg prefix)arm_joint1
        - $(arg prefix)arm_joint2
        - $(arg prefix)arm_joint3
        - $(arg prefix)arm_joint4
        - $(arg prefix)arm_joint5
        - $(arg prefix)arm_joint6
        - $(arg prefix)arm_joint7





